lib/evaluate/expression.cc

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 #include "expression.h"
+#include "variable.h"
 #include "../common/idioms.h"
+#include "../parser/characters.h"
```

Comment:
Should characters.h move to common eventually?

Reply:
It could, yes.